Error on build

I get the following error when building the jars.

./gradlew jar                                                                                                       1 ↵

> Task :client:compileJava FAILED
/Users/cjaentsch/KassenSichV/Fiskaly/fiskaly-kassensichv-client-java/client/src/main/java/com/fiskaly/kassensichv/client/authentication/TokenManager.java:59: error: cannot infer type arguments for TypeReference<T>
                new TypeReference<>() {};
                                 ^
  reason: cannot use '<>' with anonymous inner classes
  where T is a type-variable:
    T extends Object declared in class TypeReference
Note: /Users/cjaentsch/KassenSichV/Fiskaly/fiskaly-kassensichv-client-java/client/src/main/java/com/fiskaly/kassensichv/client/interceptors/TransactionInterceptor.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
1 error

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':client:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 1s
3 actionable tasks: 1 executed, 2 up-to-date


------------------------------

This seems to be easily fixed in TokenManager.java:L59

final TypeReference<HashMap<String, String>> typeReference =
                new TypeReference<HashMap<String, String>>() {};

--- a/client/src/main/java/com/fiskaly/kassensichv/client/authentication/TokenManager.java
+++ b/client/src/main/java/com/fiskaly/kassensichv/client/authentication/TokenManager.java
@@ -56,7 +56,7 @@ public class TokenManager implements Runnable {
      */
     private void setTokenPairFromResponseBody(String responseBody) throws IOException {
         final TypeReference<HashMap<String, String>> typeReference =
-                new TypeReference<>() {};
+                new TypeReference<HashMap<String, String>>() {};
 
         Map<String, String> decodedBody = mapper.readValue(responseBody, typeReference);
 

